userspace/libsinsp/parsers.cpp

```diff
@@ -4499,6 +4500,10 @@ void sinsp_parser::parse_container_json_evt(sinsp_evt *evt)
 		if(!name.isNull() && name.isConvertibleTo(Json::stringValue))
 		{
 			container_info.m_name = name.asString();
+			if(container_info.m_type == CT_DOCKER && container_info.m_name.find("k8s_POD") == 0)
```

Comment:
can we serialize the is_pod_sandbox flag instead of duplicating the logic here? it's not that simple for cri and can't be determined from the container name

Reply:
Sure.